library/public/interactor.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * @class   interactor
+ * @brief   Class used to control interaction and animation
+ *
+ * A class to control interaction with the window as well as animation.
+ * It also provide a timer callback if needed and can record/play interaction file.
+ */
+
+#ifndef f3d_interactor_h
+#define f3d_interactor_h
+
+#include "export.h"
+
+#include <functional>
+#include <string>
+#include <vector>
```

Comment:
You probably need to add `<utility>` for standard pair.

Reply:
indeed !

Reply:
Adressed in https://github.com/f3d-app/f3d/pull/355